spikes/pikepdf-write/content_tagger.py

```
@@ -26,13 +26,132 @@
+            continue
+        obj_num = xo.objgen[0] if xo.is_indirect else None
+        if obj_num is not None and obj_num in processed_forms:
+            continue
+        if obj_num is not None:
+            processed_forms.add(obj_num)
+
+        # Strip existing marked content from the form's stream.
+        try:
+            ops = list(pikepdf.parse_content_stream(xo))
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)